code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -84,6 +84,9 @@
 	opened = FALSE
 	return TRUE
 
+/obj/structure/closet/crate/shove_impact(mob/living/target, mob/living/attacker)
+	return FALSE
+
```

Comment:
Please remove this one, there's already a definition for `/obj/structure/closet/crate/shove_impact` later in this file.